pbcore/io/FastqIO.py

```diff
@@ -72,7 +72,7 @@ def __init__(self, header, sequence, quality=None, qualityString=None):
             # Only one of quality, qualityString should be provided
             assert (quality is None) != (qualityString is None)
             if quality is not None:
-                self._quality = quality
+                self._quality = np.array(quality).astype(np.uint8)
```

Comment:
I'd be open to having a conversion here, if the type doesn't match.  A cleaner conversion would be `np.array(quality, dtype=np.uint8)` as this should only require one copy.